config.ts

```diff
@@ -25,15 +25,22 @@ if (LDES_BASE === "") {
 if (!LDES_BASE.endsWith("/")) {
   LDES_BASE = LDES_BASE + "/";
 }
-if (!LDES_ENDPOINT.endsWith("/")) {
-  LDES_ENDPOINT = LDES_ENDPOINT + "/";
+
+process.env.BASE_URL = LDES_BASE; // required by the ldes-producer
+
+if (!LDES_FOLDER?.length) {
+  throw new Error('Please set the "LDES_FOLDER" environment variable. e.g: ldes-mow-registry');
+}
+
+if (!DATA_FOLDER?.length) {
```

Comment:
q: would /data not be a reasonable default here?
in any case, maybe it would be nice to also log this one together with ldes_folder?

Reply:
Nice catch! will fix

Reply:
I've made DATA_FOLDER  default to "/data", log it and updated the readme